stratum/hal/bin/barefoot/README.md

```diff
@@ -223,3 +223,21 @@ To start a shell session, you can use (requires Docker):
 
 Refer to the [p4runtime-shell](https://github.com/p4lang/p4runtime-shell)
 documentation for more information.
+
+## Building the Debian package (.deb)
+
+To build Debian package, use command below:
+
+```
+ bazel build //stratum/hal/bin/barefoot:stratum_bf_deb
+```
+
+The Debian package includes some necessary libraries like PI so you don't need to install these libraries into the target device.
+However, you still need to install some dependency packages such as **libboost**, **kmod**, and so on. Below are dependencies that required by this package:
+
+ - systemd
+ - kmod
+ - libssl1.1
+ - libedit2
+ - libjudydebian1
+ - libboost-thread1.62.0
```

Comment:
Is that still true? I see them listed in `depends`

Reply:
I think we should add command here which tells the user how to install the package.
as far as I know there are two ways, one is to use `apt install`, another is to use `dpkg -i`
`apt install` will try to install packages automatically
`dpkg -i` command will install all files to the system, but will show errors below:
```bash
root@d88707268ae6:/stratum# dpkg -i stratum_bf_deb.deb
Selecting previously unselected package stratum-bf.
(Reading database ... 6500 files and directories currently installed.)
Preparing to unpack stratum_bf_deb.deb ...
Unpacking stratum-bf (0.0.1) ...
dpkg: dependency problems prevent configuration of stratum-bf:
 stratum-bf depends on systemd; however:
  Package systemd is not installed.
 stratum-bf depends on kmod; however:
  Package kmod is not installed.
 stratum-bf depends on libssl1.1; however:
  Package libssl1.1 is not installed.
 stratum-bf depends on libedit2; however:
  Package libedit2 is not installed.
 stratum-bf depends on libjudydebian1; however:
  Package libjudydebian1 is not installed.
 stratum-bf depends on libboost-thread1.62.0; however:
  Package libboost-thread1.62.0 is not installed.

dpkg: error processing package stratum-bf (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 stratum-bf
root@d88707268ae6:/stratum# echo $?
1
root@d88707268ae6:/stratum#
```

Reply:
Yes, `dpkg` just unpacks files. You have to run `apt-get install -f` afterwards to install the dependencies. Or just use `apt-get` in the first place. My point is, that this list will get outdated and only encourages people to manually install Stratum.

Reply:
I think you found a good trade off here.